app/controllers/observations_controller.rb

```diff
@@ -1,7 +1,8 @@
 class ObservationsController < ApplicationController
   before_action :authenticate_user!
   before_action :find_proposal
-  before_action ->{authorize @proposal, :can_show!}
+  before_action ->{authorize Observation.new(proposal: @proposal)}, only: :create
```

Comment:
What about modifying `#observation` to provide this empty object if there's no id?